app/assets/javascripts/controllers/container_dashboard/charts/heatmap/heatmap-directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('miq.charts').directive('heatMap', ['$timeout',
+  function($timeout) {
```

Comment:
Maybe we can hold off merging until we upgrate to 2.6.0? @zeari, you could add the version change to this PR instead of the directives.

Though, @jeff-phillips-18, does angular-patternfly 2.6 actually depend on patternfly 2.6 itself? It usually takes @skateman some time to do the sass version, but if we can use angular-patternfly 2.6 with patterfly 2.5, there's no reason to add these directives to manageiq.